content/pt-br/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,392 @@
+
+## Controlador de Nós
+
+O {{< glossary_tooltip text="controlador" term_id="controller" >}} de nós é um componente do plano de controle do Kubernetes que gerencia vários aspectos dos nós.
+
+O controlador de nó tem várias funções na vida útil de um nó. O primeiro é atribuir um bloco CIDR ao nó quando ele é registrado (se a atribuição CIDR estiver ativada).
+
+O segundo é manter a lista interna de nós do controlador de nós atualizada com a lista de máquinas disponíveis do provedor de nuvem. Ao ser executado em um ambiente de nuvem e sempre que um nó não é saudável, o controlador de nó pergunta ao provedor de nuvem se a VM desse nó ainda está disponível. Caso contrário, o controlador de nós exclui o nó de sua lista de nós.
+
+O terceiro é monitorar a saúde dos nós. O controlador do nó é responsável por:
+
+- No caso de um nó se tornar inacessível, atualizando a condição NodeReady de dentro do `.status` do nó. Nesse caso, o controlador do nó define a condição de pronto (`NodeReady`) como condição desconhecida (`ConditionUnknown`).
+- Se um nó permanecer inacessível: será iniciado o [despejo pela API](/docs/concepts/scheduling-eviction/api-eviction/) para todos os Pods no nó inacessível. Por padrão, o controlador do nó espera 5 minutos entre marcar o nó como condição desconhecida (`ConditionUnknown`) e enviar a primeira solicitação de despejo.
+
+O controlador de nó verifica o estado de cada nó a cada `--node-monitor-period` segundos.
+
+### Limites de taxa de despejo
+
+Na maioria dos casos, o controlador de nós limita a taxa de despejo a `--node-eviction-rate` (0,1 por padrão) por segundo, o que significa que ele não despejará pods de mais de 1 nó por 10 segundos.
+
+O comportamento de despejo do nó muda quando um nó em uma determinada zona de disponibilidade se torna não saudável. O controlador de nós verifica qual porcentagem de nós na zona não são saudáveis (a condição `NodeReady` é desconhecida `ConditionUnknown` ou falsa `ConditionFalse`) ao mesmo tempo:
+
+- Se a fração de nós não saudáveis for ao menos `--unhealthy-zone-threshold` (padrão 0,55), então a taxa de despejo será reduzida.
+- Se o cluster for pequeno (ou seja, tiver menos ou igual a nós `--large-cluster-size-threshold` - padrão 50), então os despejos serão interrompidos.
+- Caso contrário, a taxa de despejo é reduzida para `--secondary-node-eviction-rate` de despejo de nós secundários (padrão 0,01) por segundo.
+
+A razão pela qual essas políticas são implementadas por zona de disponibilidade é porque uma zona de disponibilidade pode ser particionada a iniciar do plano de controle, enquanto as outras permanecem conectadas. Se o seu cluster não abranger várias zonas de disponibilidade de provedores de nuvem, o mecanismo de despejo não levará em conta a indisponibilidade por zona.
```

Comment:
I believe that in this context the sentence "become partitioned from the control plane" had the meaning of an availability zone getting disconnected from the control plane. Maybe we can reword this bit to sound more direct?